pkg/skaffold/runner/dev.go

```diff
@@ -150,5 +166,5 @@ func (r *SkaffoldRunner) Dev(ctx context.Context, out io.Writer, artifacts []*la
 		return errors.Wrap(err, "starting forwarder manager")
 	}
 
-	return r.Watcher.Run(ctx, out, onChange)
+	return r.Listen(ctx, out, onChange)
```

Comment:
Could the function be called `WaitForChanges` or `WaitForTrigger` or something that better tels what it's doing?

Reply:
yep good idea. I changed this to `WatchForChanges` and the previous method with that name to `LogWatchToUser`, WDYT?